src/compute-client/src/controller.rs

```diff
@@ -1305,6 +1305,7 @@ where
     ///     Once command history becomes durable we can remove this method and use the normal
     ///     `allow_compaction`.
     #[tracing::instrument(level = "debug", skip(self))]
+    #[allow(unused)]
     async fn allow_compaction_unvalidated(
```

Comment:
@jkosh44 not sure what's going on here?

Reply:
That should have been deleted, https://github.com/MaterializeInc/materialize/pull/14935.